File: Python/RemoveVowels.py
"""
Given a string S, remove the vowels 'a', 'e', 'i', 'o', and 'u' from it, and return the new string.

 

Example 1:

Input: "leetcodeisacommunityforcoders"
Output: "ltcdscmmntyfrcdrs"
Example 2:

Input: "aeiou"
Output: ""
 

Note:

S consists of lowercase English letters only.
1 <= S.length <= 1000

"""

class Solution:
    def removeVowels(self, S: str) -> str:  
       return S.replace('a','').replace('e','').replace('i','').replace('o','').replace('u','')

File: C++/RemoveVowels.cpp
/*

Given a string S, remove the vowels 'a', 'e', 'i', 'o', and 'u' from it, and return the new string.

 

Example 1:

Input: "leetcodeisacommunityforcoders"
Output: "ltcdscmmntyfrcdrs"


*/


class Solution {
public:
    string removeVowels(string S) {
        
        string result = "";
        
        for(auto x: S){
            if(x == 'a' || x == 'e' || x == 'i' || x == 'o' || x == 'u'){
                continue;
            
            }else{
                result += x;
            }
        }
        
        return result;
        
    }
};
